core/templates/domain/topic/subtopic.model.ts

```diff
@@ -192,7 +193,10 @@ export class Subtopic {
       skill_ids: [],
       thumbnail_filename: null,
       thumbnail_bg_color: null,
-      url_fragment: null
+      // This value can't be null as we are -passing it
```

Comment:
I do not think this reason alone makes the `null` value unneeded. And since `null` is not a substitute for an empty string, have you catered all the places where the `url_fragement` could be used? Is this a valid substitution?

Reply:
Updated PTAL